draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1580,21 +1572,43 @@ With AEGIS-128L and AEGIS-128X, random nonces can safely encrypt up to 2<sup>48<
 
 With AEGIS-256 and AEGIS-256X, random nonces can be used with no practical limits.
 
+### Key Commitment
+
+An authentication tag may verify under multiple keys, nonces, or associated data, but AEGIS is assumed to be key committing in the receiver-binding game, preventing common attacks when used with low-entropy keys such as passwords. Finding distinct keys and/or nonces that successfully verify the same `(ad, ct, tag)` tuple is expected to require ~2<sup>64</sup> attempts with a 128-bit authentication tag and ~2<sup>128</sup> attempts with a 256-bit tag.
+
+It is fully committing in the restricted setting where an adversary cannot control the associated data. As shown in {{IR23}}, with the ability to alter the associated data, it is possible to efficiently find multiple keys that will verify the same authenticated ciphertext.
+
+Protocols mandating a fully committing scheme without that restriction can provide the associated data as input to a cryptographic hash function and use the output as the `ad` parameter of the `Encrypt` and `Decrypt` functions. The selected hash function must ensure a minimum of 128-bit preimage resistance. An instance of such a function is SHA-256 {{!RFC6234}}.
+
+### Other Uses of AEGIS
+
+All variants can be used as a MAC by calling the `Encrypt()` function with the message as the `ad` and leaving `msg` empty, resulting in just a tag. However, they MUST NOT be used as a hash function; if the key is known, inputs generating state collisions can easily be crafted. Similarly, as opposed to hash-based MACs, tags MUST NOT be used for key derivation as there is no proof they are uniformly random.
+
+### Multi-User Security
```

Comment:
This could be moved under `Key and Nonce Selection` since it's also about the nonce.

Reply:
I'm not sure I follow... this is about using AEGIS as a MAC, and doesn't say anything about nonces?

Reply:
I meant `Multi-User Security`. Sorry it's not very clear.

Reply:
Oh, you referred to the "Multi-User Security" subsection.

I don't know. I kinda like having it as a dedicated subsection. The "Key and Nonce Selection" is important to every application, while the content of that section is really only relevant to applications that really care about multi-user security.

Reply:
I agree, I mean keeping the `Multi-User Security` section but repositioning it under `Key and Nonce Selection`.